### PR TITLE
Cookbook Recipe: Git-based deployment with Dokku 

### PR DESCRIPTION
**Pull request by alancwoo**
Hello! I'm submitting a cookbook recipe for deploying Kirby with git and dokku. Hope it is helpful!

### CODE REVIEW

**Comment by texnixe** [comment]
Ah cool! I always wanted to try this out myself, with your recipe it will be easier than ever. Looking forward to reviewing it!

File: content/docs/2_cookbook/9_setup/0_git-based-deployment-with-dokku/cookbook-recipe.txt [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+git remote add live dokku@IP_OF_SERVER:live
+```
+
+Amazingly, Dokku autodetects our app and knows to use the (link: https://elements.heroku.com/buildpacks/heroku/heroku-buildpack-php text: Heroku PHP Buildpack) automatically, so we can already go ahead and deploy our app. If you need more control, you will want to create a `Procfile` in the root of your repository and have a look at the (link: https://dokku.com/docs/deployment/builders/dockerfiles/#procfiles-and-multiple-processes text: documentation).
+
+Let's start by deploying our `live` app:
+
+```bash
+# LOCAL
+git push live
```
Comment (texnixe) [inline content/docs/2_cookbook/9_setup/0_git-based-deployment-with-dokku/cookbook-recipe.txt:170]: For me this only worked with
```
git push --set-upstream live main
```

**Review by texnixe — CHANGES_REQUESTED** [review]
I created a PR with a few typo fixes over at your fork. 

Apart from that and the one comment here, everything worked fine (apart from the Let's Encrypt part which is likely due to the domain not having propagated yet).

Great recipe! I particularly like the bonus part.

**Comment by alancwoo** [comment]
Great, thanks for the fixes @texnixe - hope it's useful!

**Pull request closed without merging**